SpatialConnect/SCBackendService.m

```diff
@@ -325,10 +324,14 @@ - (void)authListener {
 
   [authed subscribeNext:^(NSNumber *n) {
     [self connect];
-    [[[connectedToBroker filter:^BOOL(NSNumber *n) {
+    [[connectedToBroker filter:^BOOL(NSNumber *n) {
       return n.boolValue;
-    }] take:1] subscribeNext:^(id x) {
-      [self registerAndFetchConfig];
+    }] subscribeNext:^(id x) {
```

Comment:
Instead of subscribing in a subscribe:
`[[authed flattenMap:^RACSignal*(id x) {
    [self connect]
    return connectedToBroker;
}] filter:^BOOL(NSNumber *n) {
   return n.boolValue;
}]`